doc/tutorial/single_cell_bluepyopt.rst

```diff
@@ -0,0 +1,288 @@
+
+.. figure:: single_cell_bluepyopt_gui_l5pc_bAP_attempt.png
+    :width: 800
+    :align: center
+
+BluePyOpt solves this issue with the Neuron simulator by choosing a particular section on the apical dendrite. While a similar approach is possible in Arbor using the :func:`arbor.cable_cell.locations` method, for reproducibility it is preferable to determine the location directly in the Arbor GUI.
+
+For this purpose, we can use the identify a distal apical branch (or segment) by hovering with the mouse over the morphology and keeping a focus on the data in the left lower corner. We can then define a region with 
+
+.. code-block::
+
+   (proximal-interval (distal (branch <branch-id>)))
+ 
+that includes the path (Region C, yellow) from that branch to the (apical) root (Locset D, blue). Restricting the above ``distal-translate``-set of locations to this region evaluates to a unique concrete location for the probe (Locset F, red). 
+
+.. figure:: single_cell_bluepyopt_gui_l5pc_bAP_final.png
+    :width: 800
+    :align: center
+
+The final expression for the probe location on the morphology with axon-replacement, hence, is
+
+.. code-block::
+
+   (restrict
+      (distal-translate
+         (proximal (region "apic"))
+         660)
+      (proximal-interval
+         (distal (branch 123))))
+
+For the following, we will add a label for this location as well as for the center of the soma to the label dictionary.
+
+.. literalinclude:: ../../python/example/single_cell_bluepyopt_l5pc.py
+   :language: python
+   :dedent:
+   :lines: 12,24-26
+
+
+The cable cell and protocols
+----------------------------
+
+Following BluePyOpt's layer-5 pyramidal cell example, we have to define protocols with stimuli and probes as well as make the appropriate mechanism catalogues available to the recipe.
+
+To replicate the ``bAP`` protocol in BluePyOpt, we add a short stimulus centrally at the soma to the decor.
+
+.. literalinclude:: ../../python/example/single_cell_bluepyopt_l5pc.py
+   :language: python
+   :dedent:
+   :lines: 28,30-42
+
+The decor allows to further customize the discretization for a subsequent simulation using :func:`arbor.decor.discretization` as examined in the :ref:`tutorial for a detailed single cell model <tutorialsinglecellswc>`. For brevity, we will use the default here except on the soma. As a next step, we create an :class:`arbor.cable_cell` using the morphology, label dictionary and decor.
+
+Then, we can install a voltage probe on the apical dendrite.
+
+.. literalinclude:: ../../python/example/single_cell_bluepyopt_l5pc.py
+   :language: python
+   :dedent:
+   :lines: 44-46
+
+
+To prepare the cable cell for simulation, we can either construct an instance of the :class:`arbor.single_cell_model` from the cable cell (shown in the simple-cell example) or - for more complicated use cases - define our own recipe. We will follow the latter for the layer-5 pyramidal cell. 
```

Comment:
In this example I don't see a reason for creating your own `recipe`. Am I missing it?

Reply:
The recipe is simply there as a starting point for more customization. The L5PC tutorial would also work with just a `single_cell_model`, but then there's no added value over what's in the simple-cell tutorial.